test/performance/io/format_vienna_benchmark.cpp

```diff
@@ -0,0 +1,149 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#if __has_include(<seqan/rna_io.h>)
+    #include <seqan/rna_io.h>
+#endif
+
+#include <sstream>
+#include <string>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/io/structure_file/input.hpp>
+#include <seqan3/io/structure_file/output.hpp>
+#include <seqan3/io/structure_file/format_vienna.hpp>
+#include <seqan3/test/performance/units.hpp>
+
+using namespace seqan3;
+using namespace seqan3::test;
+
+inline constexpr size_t iterations_per_run = 1024;
+
+inline std::string const header{"seq foobar blobber"};
+inline std::string const sequence
```

Comment:
We have the utility `generate_sequence` in `test/include/seqan3/test/performance/sequence_generator`

```cpp
#include <seqan3/test/performance/sequence_generator.hpp>
...
auto dna4_seq seq = generate_sequence<dna4>(500 /*length*/, 0/*variance of uniform dist*/, 0 /*seed*/)
                  | seqan3::views::to_char 
                  | seqan3::to<std::string>;
```
:)

Reply:
with views::persist this works well now...